frontend/app/src/composables/history/events/notes.ts

```diff
@@ -119,10 +119,22 @@ export function useHistoryEventNote(): UseHistoryEventsNoteReturn {
     if (get(scrambleData) && counterpartyVal === 'monerium')
       notesVal = findAndScrambleIBAN(notesVal);
 
-    // label each word from notes whether it is an address or not
     const words = notesVal.split(/[\s,]+/);
+    const assetWords = asset ? asset.split(/\s+/) : [];
+    const processedWords: string[] = [];
+
+    for (let i = 0; i < words.length; i++) {
```

Comment:
Can we add a test for this @lukicenturi ? just to make sure we don't accidentally break it in the future?